README.md

```diff
@@ -83,3 +83,4 @@ Gracias a su legado como el héroe de Hyrule, cada encarnación de Link recibe d
 
 
 
+Jiménez Treviño Emilio Cristóbal
```

Comment:
Te faltó agregar de qué generación eres.

Reply:
te faltó tal cosa.
Esto podría causar errores.

Reply:
Done in https://github.com/PROTECO/git_gihub_gen43/pull/3/commits/45e1ee551deca86dd4775358709ed04784b96045

Reply:
Done in 
https://github.com/PROTECO/git_gihub_gen43/pull/3/commits/45e1ee551deca86dd4775358709ed04784b96045